packages/siop-oid4vp/lib/__tests__/AuthenticationRequest.request.spec.ts

```diff
@@ -268,10 +362,10 @@ describe('create Request Uri should', () => {
     const uriRequest = await URI.fromOpts(opts)
 
     const uriDecoded = decodeURIComponent(uriRequest.encodedUri)
-    expect(uriDecoded).toContain(`openid://?request=eyJhbGciOi`)
+    expect(uriDecoded).toContain(`openid4vp://?client_id=https://www.example.com/`)
```

Comment:
Please do not remove the test for request = jwt. This obviously should have been a new expect. And then not toContain but a check that it starts with

Reply:
fixed